services/realtime/.env.example

```diff
@@ -25,9 +25,9 @@ SECRET_FOR_SUMMARY=my_secret
 # REACTION_WRITEBACK_INTERVAL=5000
 
 # Sometimes needed for setting up a test user in local testing with RabbitMQ
-#
-# RABBITMQ_USERNAME=services/realtime
-# RABBITMQ_PASSWORD=1234
+# need to go to http://localhost:15672/ to give this user the authority
```

Comment:
We should add information to the README to explain how to do this:

1. Go to `http://localhost:15672/`
2. Log in with the username/password `admin`/`admin`
3. Open the Admin tab
4. Expand the Add User section if it is not already open
5. Enter the username from the env var `RABBITMQ_USERNAME`
6. Enter the password from the env var `RABBITMQ_PASSWORD`
7. Repeat / confirm the same password
8. Click Add User
9. The user does not currently have any permissions. To add permissions, click on the username in the table.
10. Under the Permissions section, change nothing and just click Set Permission. This will grant full access.